app/screens/home/search/results/results.tsx

```diff
@@ -32,36 +47,81 @@ const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
 
 type Props = {
     searchValue: string;
-    selectedTab: 'messages' | 'files';
+    selectedTab: TabType;
     currentTimezone: string;
     isTimezoneEnabled: boolean;
     posts: PostModel[];
+    fileChannels: ChannelModel[];
     fileInfos: FileInfo[];
     scrollRef: React.RefObject<FlatList>;
     onScroll: (event: NativeSyntheticEvent<NativeScrollEvent>) => void;
     scrollPaddingTop: number;
     loading: boolean;
+    canDownloadFiles: boolean;
+    publicLinkEnabled: boolean;
 }
 
 const emptyList: FileInfo[] | Array<string | PostModel> = [];
 
-const SearchResults = ({
+const Results = ({
     currentTimezone,
     fileInfos,
     isTimezoneEnabled,
     posts,
+    fileChannels,
     searchValue,
     selectedTab,
     scrollRef,
     onScroll,
     scrollPaddingTop,
     loading,
+    canDownloadFiles,
+    publicLinkEnabled,
 }: Props) => {
     const theme = useTheme();
     const paddingTop = useMemo(() => ({paddingTop: scrollPaddingTop, flexGrow: 1}), [scrollPaddingTop]);
-
     const orderedPosts = useMemo(() => selectOrderedPosts(posts, 0, false, '', '', false, isTimezoneEnabled, currentTimezone, false).reverse(), [posts]);
 
+    const isTablet = useIsTablet();
+    const galleryIdentifier = 'search-files-location';
+
+    const getContainerStyle = useMemo(() => {
+        let padding = 0;
+        if (selectedTab === TabTypes.MESSAGES) {
+            padding = posts.length ? 4 : 8;
+        } else {
+            padding = fileInfos.length ? 8 : 0;
+        }
+        return {top: padding};
+    }, [selectedTab, posts, fileInfos]);
+
+    const getChannelName = useCallback((id: string) => {
```

Comment:
You are calling this function, so there is no need for useCallback. Either do not use useCallback, or memoize the result of the function (i.e. use `useMemo`)